### PR TITLE
fix #275483: Added the ability to change the color of the playbackCursor

### PR DESCRIPTION
**Pull request by SKefalidis**
Resolves: https://musescore.org/en/node/275483

It gives the user the ability to change the color of the playback cursor (can't be made fully opaque). 
Because having a fully opaque cursor does not have an obvious use if you choose a fully opaque color you get the default transparency level, so the user can change color with even fewer clicks.

To solve the original issue you make the cursor transparent and the voice color black.

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [n/a] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

File: mscore/scoreview.h [hunk 1]
```diff
@@ -117,6 +117,8 @@ class ScoreView : public QWidget, public MuseScoreView {
 
       //--input state:
       PositionCursor* _cursor;
+      QColor _cursorColor;
+      const int maxCursorAlpha = 220;
```
Comment (igorkorsukov) [inline mscore/scoreview.h]: Better name: MAX_CURSOR_ALPHA, to immediately see that it is a constant

Comment (SKefalidis) [reply mscore/scoreview.h]: @igorkorsukov I chose lower-case because the other constant in the same file is like that. Should I change both?

Comment (igorkorsukov) [reply mscore/scoreview.h]: Only this one is better.  Others are best fixed with a separate pull request.

**Comment by Spire42** [comment]
I already have this implemented as part of my redesign of the playhead (some discussion [here](https://musescore.org/en/node/301224) in collaboration with @Tantacrul.

My work is currently in progress so there's no PR yet.

**Comment by SKefalidis** [comment]
@Spire42 Then I should close this, right?

**Comment by Spire42** [comment]
> @Spire42 Then I should close this, right?

Yes, I think that would probably be best. Sorry about that.

**Comment by SKefalidis** [comment]
Alright then :^)

**Pull request closed without merging**